static/templates/compose_pm_mentions_user_alert.hbs

```diff
@@ -0,0 +1,7 @@
+<div class="compose_pm_mentions_user_alert" data-full_name="{{full_name}}">
```

Comment:
This should be `data-full-name` as per the convention we use at other places. And use this element to get the full name in the click handler above.

Reply:
DONE. good catch, I also added data-user-id to pass that as well.